Add running docker-compose after pull from upstream to the instruction (#139)
open-telemetry/opentelemetry-dotnet-instrumentation: docs/PULL_FROM_UPSTREAM.md

    * Run unit tests, commit any needed fixes, repeat until passing unit tests
    * Update versions and integrations json by running: ` cd \build\tools\PrepareRelease && dotnet run -- versions integrations` (remember to revert wcf and other windows-only frameworks if you are using different platform)

>     * Run integration tests, commit any needed fixes, until passing integrations tests
>     * Run `docker-compose run build`, `docker-compose run Profiler` and `docker-compose run package` to verify `.sh` files are valid after the merge.

Reviewer: ```suggestion
    * Run each of the following commands and commit any needed fixes, until it passes:
        - `docker-compose run --rm build`
        - `docker-compose run --rm Profiler`
        - `docker-compose run --rm IntegrationTests`
        - `docker-compose run --rm package`
```

Author: 👍 much better

    * Run unit tests, commit any needed fixes, repeat until passing unit tests
    * Update versions and integrations json by running: ` cd \build\tools\PrepareRelease && dotnet run -- versions integrations` (remember to revert wcf and other windows-only frameworks if you are using different platform)

>     * Run integration tests, commit any needed fixes, until passing integrations tests

Reviewer: this line can be removed
